src/gsi/combine_radobs.f90

```diff
@@ -74,7 +74,7 @@ subroutine combine_radobs(mype_sub,mype_root,&
 
      nread=0
      if (mype_sub==mype_root) nread = ncounts1
-     if (ncounts1 == 0)return
+     if (ncounts1 <= 0)return
```

Comment:
I wonder if it would be worth it to change `ncounts1` to an `integer (i_llong)` to help prevent overflow?